api-server/src/server/boot/challenge.js

```diff
@@ -333,6 +341,56 @@ function backendChallengeCompleted(req, res, next) {
     .subscribe(() => {}, next);
 }
 
+function saveChallenge(req, res, next) {
+  const user = req.user;
+  const { id, challengeFiles: files = [] } = req.body;
+
+  const savableChallenges = getChallenges()
+    .filter(challenge => challenge.challengeType === 14)
+    .map(challenge => challenge.id);
+
+  if (!savableChallenges.includes(id)) {
+    return res.status(403).send('That challenge type is not savable');
+  }
+
+  let challengeToSave = {
+    id,
+    lastSavedDate: Date.now(),
+    files: files.map(file =>
+      pick(file, ['contents', 'fileKey', 'index', 'name', 'path', 'ext'])
```

Comment:
I think this needs to be `key` (server-side).

```suggestion
      pick(file, ['contents', 'key', 'index', 'name', 'path', 'ext'])
```

Reply:
Yeah, it should be due to the type of the collection.  The ajax call needs changing, too, so I'll leave a comment.